omicron/cli/process.py

```diff
@@ -645,9 +650,12 @@ def main(args=None):
                     pass
 
     if statechannel:
-        logger.debug("State channel = %s" % statechannel)
-        logger.debug("State bits = %s" % ', '.join(map(str, statebits)))
-        logger.debug("State frametype = %s" % stateft)
+        logger.debug(f"State channel {statechannel}")
+        if statebits == 'guardian':
+            logger.debug(f"State bits {statebits}")
+        else:
+            logger.debug("State bits = %s" % ', '.join(map(str, statebits)))
```

Comment:
Maybe cleaner to use something like

```suggestion
        if statebits == 'guardian':
            bitstr = "guardian"
        else:
            bitstr = ", ".join(map(str, statebits))
        logger.debug(f"State bits = {bitstr}")
```

Reply:
OK